lib/repoman/required_file_checker.rb

```diff
@@ -0,0 +1,27 @@
+module Repoman
+  class RequiredFileChecker
```

Comment:
Does this need its own tests? Or can some of the `LicenseChecker` tests be made into its tests?